frontend/src/containers/NewGroup.js

```diff
@@ -267,13 +273,15 @@ export default connect(mapStateToProps, {
   validateSchema
 })(NewGroup);
 
-export function mapStateToProps({router, form}) {
+export function mapStateToProps({router, form, groups}) {
 
+  const hostCollective = (groups && router.params.slug) ? groups[router.params.slug] : null;
```

Comment:
doesn't this mean that any `/:slug/apply` will work? We should make sure that settings.HostId is valid before this page loads.